packages/remix-react/browser.tsx

```diff
@@ -151,6 +151,16 @@ export function RemixBrowser(_props: RemixBrowserProps): ReactElement {
       );
     }
 
+    if (!window.__remixContext.future.v2_normalizeFormMethod) {
+      warnOnce(
+        false,
+        "⚠️  DEPRECATED: Please enable the `future.v2_normalizeFormMethod` flag to " +
+          "prepare for the Remix v2 release. Lowercase `useNavigation().formMethod`" +
+          "values are being normalized to uppercase in v2 to align with the `fetch()` " +
+          "behavior.  For more information, see https://remix.run/docs/hooks/use-navigation"
+      );
+    }
+
```

Comment:
ooh i didn't even think to include deprecations in the browser console

Reply:
yeah I've only done it for remix-react related flags so far.  Not yet convinced it makes sense to put say a flat routes deprecation log in the console

Reply:
looks like we also let this log in production 🙈 
![image](https://user-images.githubusercontent.com/11698668/226965259-52660bb3-754e-4e7c-b49e-d5f3a3b9df94.jpeg)

Reply:
PR to fix #5874